tools/dashboard-plugin-template/src/plugin/Plugin.tsx

```diff
@@ -25,6 +25,8 @@ export class Plugin extends DashboardPluginV1 {
     public readonly author = packageJson.author;
     public readonly displayName = packageJson.name;
     public readonly version = packageJson.version;
+    public readonly minEngineVersion = "bundled";
+    public readonly maxEngineVersion = "bundled";
```

Comment:
One thing I'm not sure about:
- how to ensure that plugin + entry point minEngineVersion / maxEngineVersion match?

Reply:
Like how to avoid having maxVersion 8.8.0 and building with 8.9.0? I am not sure either... Maybe some "quick and dirty" sh step in validate?

Reply:
Now it's taken from the entry point.